website/cue/reference/components.cue

```diff
@@ -937,7 +937,7 @@ components: {
 				}
 			}
 
-			if Kind != "source" {
+			if Kind != "source" && Kind != "transform" {
```

Comment:
We're injecting `inputs`, etc, via the new "base" files, so we don't want it to be added here for transforms, as they're the only component type migrated to the base file approach.